tests/services/test_snippet.py

```diff
@@ -27,6 +27,8 @@ class TestSnippet(metaclass=AIOTestMeta):
     async def setup(self):
         conf = get_conf(
             pkg_resources.resource_filename('xsnippet_api', 'default.conf'))
+        conf['auth'] = {'secret': 'SWORDFISH'}
```

Comment:
Hm.. Why do we need 'auth' on services layer? I thought it's required for API only. Could you please explain?